core/domain/services/admin/events/editor/EventManagers.php

```diff
@@ -39,49 +44,62 @@ public function __construct(Utilities $utilities)
      */
     public function getData(int $eventId)
     {
-        // first get a list of WP_Roles that have "event manager" capabilities
-        $event_manager_roles = $this->getEventManagerRoles();
-        // then get a list of WP Users that have any of those roles
-        $event_manager_users = $this->getEventManagerUsers($event_manager_roles);
-        // now convert to a format that's usable by GQL
-        $event_managers = [];
-        foreach ($event_manager_users as $user) {
-            $GUID             = $this->utilities->convertToGlobalId('user', $user->ID);
-            $event_managers[] = [
-                'id'   => $GUID,
-                'name' => $user->display_name,
-            ];
+        if (empty($this->event_managers)) {
+            [$roles, $capabilities] = $this->getRoleAndCapabilities();
```

Comment:
Are you trying to bring JS destructuring to PHP? 🤔 
No idea which PHP syntax is that. 🤷‍♂️

Reply:
ya that was added in PHP 7, so can only use it for EDTR related stuff for now:
https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.symmetric-array-destructuring

Reply:
Nice, thanks